src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/view/fragments/SelectResource.kt

```diff
@@ -3,115 +3,79 @@ package org.wycliffeassociates.otter.jvm.app.ui.projectcreation.view.fragments
 import org.wycliffeassociates.otter.jvm.app.ui.projectcreation.viewmodel.ProjectCreationViewModel
-import org.wycliffeassociates.otter.jvm.app.widgets.WizardCard
-import org.wycliffeassociates.otter.jvm.app.widgets.wizardcard
-import javax.annotation.Resource
+import java.io.File
 
 class SelectResource : View() {
     val viewModel: ProjectCreationViewModel by inject()
-    val toggleGroup = ToggleGroup()
 //    override val complete = viewmodel.valid(viewmodel.resource)
     override val root =  hbox(40) {
                 alignment = Pos.CENTER
-//                wizardcard {
-//                    text = "Bible"
-//                    selected = true
-//                    // image = imageLoader(File("/Users/NathanShanko/Downloads/OBS.svg"))
-//                }
                 togglegroup {
 
-                    addEventHandler(ActionEvent.ACTION) {
-                    }
-
-                    togglebutton {
-                        contentDisplay = ContentDisplay.TOP
-                        graphic = MaterialIconView(MaterialIcon.COLLECTIONS_BOOKMARK)
-                        if (isSelected) {
-                            addClass(ResourceStyles.selectedCard)
-                        } else {
-                            addClass(ResourceStyles.unselectedCard)
-                        }
-                        text = messages["bible"]
-                        alignment = Pos.CENTER
-
-                        selectedProperty().onChange {
-                            if (it) {
-                                removeClass(ResourceStyles.unselectedCard)
-                                addClass(ResourceStyles.selectedCard)
-                            } else {
-                                removeClass(ResourceStyles.selectedCard)
-                                addClass(ResourceStyles.unselectedCard)
-                            }
-                        }
-
-                    }
-
-                    togglebutton {
-                        contentDisplay = ContentDisplay.TOP
-                        //graphic = imageLoader(File("/Users/NathanShanko/Downloads/OBS.svg"))
-                        if (isSelected) {
-                            addClass(ResourceStyles.selectedCard)
-                        } else {
-                            addClass(ResourceStyles.unselectedCard)
-                        }
-                        text = messages["obs"]
-                        alignment = Pos.CENTER
-                        selectedProperty().onChange {
-                            if (it) {
-                                removeClass(ResourceStyles.unselectedCard)
-                                addClass(ResourceStyles.selectedCard)
-                            } else {
-                                removeClass(ResourceStyles.selectedCard)
-                                addClass(ResourceStyles.unselectedCard)
+                    viewModel.resourceList.onChange {
```

Comment:
Does the toggle group need cleared first if the resourceList has existing items?

Reply:
no. i don't think so. the call to get the resources only happens once initially when the wizard is initialized. and since the response is a single it shouldn't push out any more changes after the initial one i believe